src/cms/views/dashboard/region_selection.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.views.generic import TemplateView
+
+
+@method_decorator(login_required, name="dispatch")
+class RegionSelection(TemplateView):
+    """
+    View for the region selection
+    """
+
+    #: The template to render (see :class:`~django.views.generic.base.TemplateResponseMixin`)
+    template_name = "dashboard/region_selection.html"
+
+    def get(self, request, *args, **kwargs):
+        """
+        Render the region selection view.
+
+        :param request: Object representing the user call
+        :type request: ~django.http.HttpRequest
+
+        :param args: The supplied arguments
+        :type args: list
+
+        :param kwargs: The supplied keyword arguments
+        :type kwargs: dict
+
+        :return: The rendered template response
+        :rtype: ~django.template.response.TemplateResponse
+        """
+        user = request.user
+        regions = user.profile.regions
+        return render(
+            request,
+            self.template_name,
+            {
+                "user_regions": regions.all(),
+                **self.get_context_data(**kwargs),
+            },
+        )
```

Comment:
```suggestion
```
Since this template view doesn't need extra context (see my comment above), you can just omit this method altogether and rely on the inbuilt Django [TemplateView](https://docs.djangoproject.com/en/3.2/ref/class-based-views/base/#django.views.generic.base.TemplateView) for rendering the request.